json/src/fs2/data/json/internal/package.scala

```diff
@@ -23,6 +23,9 @@ package object internals {
 
   private[json] type Result[F[_], In, Out] = Option[(Chunk[In], Int, Stream[F, In], Out)]
 
+  private[json] def emitChunk[T](chunkAcc: List[T]) =
```

Comment:
Not sure that would make any difference, but maybe add `@inline`?

Reply:
Might be, this is part of a broader task with benchmarking to see if it does, and the library is not compiled with the optimizer anway.